packages/api/src/controllers/organicHistoryController.js

```diff
@@ -13,118 +13,20 @@
  *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
  */
 
-const baseController = require('./baseController');
-const { transaction, Model } = require('objection');
-const model = require('../models/organicHistoryModel');
-const locationModel = require('../models/locationModel');
+
+const organicHistoryModel = require('../models/organicHistoryModel');
 
 module.exports = {
-  async addEntry(req, res) {
-    if (!locationModel.isCropEnabled(req.body.location_id)) {
-      return res.status(400).send('Location must be crop enabled.');
-    }
 
-    const trx = await transaction.start(Model.knex());
-    try {
-      const result = await baseController.postWithResponse(model, req.body, req, { trx });
```

Comment:
baseController should be deprecated and const trx = await transaction.start(Model.knex()); syntax causes error when await trx.commit(); or await trx.rollback(); are missing

Reply:
Model.transaction() is better
https://vincit.github.io/objection.js/guide/transactions.html#creating-a-transaction